flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/model/SequenceNumber.java

```diff
@@ -0,0 +1,87 @@
+package org.apache.flink.streaming.connectors.kinesis.model;
```

Comment:
Did you do "mvn verify" or another phase? I think the plugin is executed on verify